backend/resolver/aws/aws.go

```diff
@@ -168,10 +168,15 @@ func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (
 		if err != nil {
 			return nil, err
 		}
+
 		if ok {
-			return r.instanceResults(ctx, patternValues["account"], patternValues["region"], []string{patternValues["instance_id"]}, limit)
+			// in the case of account not found, we continue to the below flow of only being concerned with the ID, rather than the account and region
+			if _, ok := r.client.AccountsAndRegions()[(patternValues["account"])]; ok {
```

Comment:
nit: redundant parenthesis
```suggestion
			if _, ok := r.client.AccountsAndRegions()[patternValues["account"]]; ok {
```

but you could also assign to a local var which would improve readability